testing/mod.ts

```diff
@@ -6,11 +6,18 @@ interface Constructor {
   new (...args: any[]): any;
 }
 
+export class AssertionError extends Error {
```

Comment:
Is this needed to avoid the stack trace?

Reply:
No, good point this is distinct. Initially when I was debugging something it wasn't immediately clear whether or not it was another Error or from the assert, this change disambiguates.

Reply:
Can we also show which assert failed?

Reply:
in a separate PR. this one is just a one line change.